images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -687,3 +701,23 @@ function Get-HashFromGitHubReleaseBody {
     }
     return $result
 }
+function Test-File-Signature {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$FilePath,
+        [Parameter(Mandatory=$true)]
+        [string]$ExpectedThumbprint
+    )
+ 
+    $signature = Get-AuthenticodeSignature $FilePath
+ 
+    if ($signature.Status -ne "Valid") {
+        throw "Signature status is not valid"
+    }
+
+    if ($signature.SignerCertificate.Thumbprint.Contains($ExpectedThumbprint) -ne $true) {
+        throw "Signature thumbprint do not match expected"
+    }
+
+    Write-Output "Signature is valid"
```

Comment:
This message doesn't look informative. Let's either get rid of it or output more information (e.g. name of the file)

Reply:
Add file pathto Write-Output message.